hosting/templates/hosting/base_short.html

```diff
@@ -48,7 +51,7 @@
 </head>
 
 <body>
-
+    {% cms_toolbar %}
```

Comment:
Why do we need this one here?

Reply:
This is for the cms toolbar, which will be needed to add the calculator plugin. Due to a known issue in the cms_toolbar template tag, it cannot be inside a `block` tag and so has to be added in the base template itself. There is a possible workaround using `include` tag, but that didn't seem to work with the current template structure. So decided to go with the simple solution.